tools/quickstart/quickstart/docker.py

```diff
@@ -189,7 +190,8 @@ def get_optional_docker_service_names(base_dir) -> List[str]:
 
     if is_bridge_prepared(base_dir=base_dir):
         docker_service_names.append("bridge-client")
-        docker_service_names.append("foreign-node")
+        if is_bridge_using_default_json_rpc(base_dir):
```

Comment:
Is there no better way? 
What if someone choose to run there node under the same url?

Reply:
The other way would be to ask whether they want to use their own URL outside of `bridge.setup_interactively` or return this information from `bridge.setup_interactively` so that we can give it to `docker.update_and_start` but I do not know if it is better as we still want to ask whether they want to use their own URL after we ask whether they want to set up the bridge.